modules/core/src/main/java/org/apache/fluo/core/client/FluoAdminImpl.java

```diff
@@ -76,6 +77,28 @@
 
   private final String appRootDir;
 
+  /**
+   * Kerberos autentication method.
+   * 
+   * @param realm Realm to be used in authentication.
+   * @param keytab Keytab path.
+   * @since 1.3.0
+   */
+  public void loginWithKerberos(final String realm, final String keytab) {
```

Comment:
As far as I can tell, this only enables authentication to HDFS using HDFS client-side authentication features. The name should indicate `hdfsLoginWithKerberos` or some other indication. At the very least, the Javadoc should indicate that this is HDFS authentication.

Reply:
As we notice, just the jar copy process demanded kerberos authentication. All other components were able to retrieve the tokens from environment. But I agree with you. Fluo should work using the environment authentication. 

I'll try other approaches. For while if someone want to use Fluo on kerberos this could help.

Reply:
Oh, interesting. I'm surprised other components worked from the environment, but this did not. I wonder why. Did you try fluo-yarn? Also, do you know if this change allows it to work with Accumulo with Kerberos tokens? Or did you only try Accumulo using regular PasswordTokens?

Reply:
Another possible way to fix this would be to ensure we use hadoop config from the classpath and document that.

Reply:
Yes! Our Accumulo is kerberized too. We have no problem with. We didn’t tried fluo-yarn yet on this environment. We can try.